### PR TITLE
Update static analysis image to include java 8 and 11 runtimes.

### PR DESCRIPTION
**Pull request by dzlier-gcp**
Fixes #issue

> It's a good idea to open an issue first for discussion.

- [x] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md)
- [x] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**

### CODE REVIEW

File: .kokoro/static_analysis/common.cfg [hunk 1]
```diff
@@ -19,13 +19,19 @@ gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/trampoline"
...
     key: "TRAMPOLINE_BUILD_FILE"
     value: "github/java-docs-samples/.kokoro/tests/run_static_analysis.sh"
 }
 
+# Set the JAVA VERSION env var.
+env_vars: {
+    key: "JAVA_VERSION"
+    value: "1.8,11"
+}
+
```
Comment (kurtisvg) [inline .kokoro/static_analysis/common.cfg:34]: The static analysis check doesn't use this: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/.kokoro/tests/run_static_analysis.sh

**Review by kurtisvg — CHANGES_REQUESTED** [review]

**Review by lesv — COMMENTED** [review]
@dzlier-gcp This should have been discussed at the Java meeting today.

**Comment by dzlier-gcp** [comment]
> @dzlier-gcp This should have been discussed at the Java meeting today.

I didn't think I'd be working in this repo, I just noticed that static analysis has been failing while I was resolving a Healthcare Datasets bug.

Should I close this for now for further discussion, or just remove the unused env var?

File: .kokoro/static_analysis/common.cfg (hunk 1 above)
Comment (dzlier-gcp) [reply .kokoro/static_analysis/common.cfg:34]: Maven uses this though doesn't it?

Comment (dzlier-gcp) [reply .kokoro/static_analysis/common.cfg:34]: Or I guess static analysis doesn't run any mvn commands.

Comment (kurtisvg) [reply .kokoro/static_analysis/common.cfg:34]: Maven doesn't, there is something in the other script we used to control which versions are run:
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/a683a732cff3d6833815461c2d59bf3f64c5c0a6/.kokoro/tests/run_tests.sh#L159-L164

**Review by kurtisvg — APPROVED** [review]

**Comment by kurtisvg** [comment]
This is only controlling which JVM version the static analysis check is being run with. It seems fine to run it on Java 11 over Java 8, since some samples only support Java 11 or higher. This seems okay to merge unless I'm missing smething. 

It's likely still going to fail for a majority of the samples, as we discussed in the Java maintenance meeting the checks are overly aggressive (which Les is looking into).

**Comment by dzlier-gcp** [comment]
> This is only controlling which JVM version the static analysis check is being run with. It seems fine to run it on Java 11 over Java 8, since some samples only support Java 11 or higher. This seems okay to merge unless I'm missing smething.
> 
> It's likely still going to fail for a majority of the samples, as we discussed in the Java maintenance meeting the checks are overly aggressive (which Les is looking into).

All right, I need a new approval after removing that env var though.

**Comment by dzlier-gcp** [comment]
Oh nvm

**Pull request merged**